pkg/ingress/controller/openstack/octavia.go

```diff
@@ -455,37 +455,44 @@ func (os *OpenStack) EnsurePoolMembers(deleted bool, poolName string, lbID strin
 		return nil, fmt.Errorf("error waiting for loadbalancer %s to be active: %v", lbID, err)
 	}
 
-	// Batch update pool members
-	var members []pools.BatchUpdateMemberOpts
-	for _, node := range nodes {
-		addr, err := getNodeAddressForLB(node)
+	if os.config.Octavia.UseLegacyAPICalls {
+		// Serially update pool members
+		err = openstackutil.SeriallyReconcilePoolMembers(os.Octavia, pool, *nodePort, lbID, nodes)
 		if err != nil {
-			// Node failure, do not create member
-			logger.WithFields(log.Fields{"nodeName": node.Name, "error": err}).Warn("failed to create LB pool member for node")
-			continue
+			return nil, fmt.Errorf("error reconciling pool members for pool %s: %v", pool.ID, err)
```

Comment:
so... random idea, if you do `return &pool.ID, nil` here, I think you don't have to change the indent of the following bits, which should help when we merge changes in from upstream?

Reply:
Nice idea @JohnGarbutt - applied the same process in a couple of places and it makes the diff much smaller!

Will test.

Reply:
Works nicely